staging/src/k8s.io/apiserver/pkg/endpoints/audit_test.go

```diff
@@ -326,6 +337,9 @@ func TestAudit(t *testing.T) {
 					t.Errorf("[%s,%d] %v", test.desc, i, err)
 				}
 			}
+			if err := requestUserAgentMatches(0, userAgent)(events); err != nil {
```

Comment:
I think all events should have the user agent set, not only the first event.

Reply:
Did not look too much, addressed now